ToolBox/CoinApi/CoinApiSymbolMapper.cs

```diff
@@ -31,7 +33,8 @@ public class CoinApiSymbolMapper : ISymbolMapper
     {
         private const string RestUrl = "https://rest.coinapi.io";
         private readonly string _apiKey = Config.Get("coinapi-api-key");
-
+        private readonly bool _useLocalSymbolList = Config.GetBool("coinapi-use-local-symbol-list", false);
+        private readonly FileInfo _coinApiSymbolsListFile = new FileInfo("CoinApiSymbols.json");
```

Comment:
Should this file be sourced from configuration instead of hard-coded?

Reply:
Agree we should give this option with a default value:
``` C#
private readonly FileInfo _coinApiSymbolsListFile = new FileInfo(
    Config.Get("coinapi-default-symbol-list-file", "CoinApiSymbols.json"));
```